requirements.txt

```diff
@@ -19,7 +19,7 @@ Sphinx
 pytest
 expecttest
 parameterized
-torchdata
+torchdata>0.5
```

Comment:
You might want to capture in your release runbook to update this version per release.

Reply:
Done